Patch created by brosx "diffPackage" tool

usage:
    diffPackage <tarName> <pkgName> <outName> <patchName> <files>

actual:
    diffPackage "file-5.34.tar.gz" "file" "host-file" "copy-magic-libs-needed-for-file-linkage" "\
        configure.ac \
        src/Makefile.am \
    ";


===============================================================================
--- a/configure.ac	2018-07-25 07:49:48.000000000 +0200
+++ b/configure.ac	2019-01-02 12:55:39.000000000 +0100
@@ -1,6 +1,6 @@
 dnl Process this file with autoconf to produce a configure script.
 AC_INIT([file],[5.34],[christos@astron.com])
-AM_INIT_AUTOMAKE([subdir-objects foreign])
+AM_INIT_AUTOMAKE
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
 AC_CONFIG_HEADERS([config.h])
===============================================================================
--- a/src/Makefile.am	2017-11-03 00:26:20.000000000 +0100
+++ b/src/Makefile.am	2019-01-02 12:55:06.000000000 +0100
@@ -4,6 +4,17 @@
 
 bin_PROGRAMS = file
 
+.lib/file: copy_magic_lib
+
+all-am: copy_magic_lib
+
+.PHONY: copy_magic_lib
+copy_magic_lib: libmagic.la
+	if [ -f .host-build-marker ]; then \
+		cp .libs/libmagic.1.dylib $(libdir)/libmagic.1.dylib; \
+		cp .libs/libmagic.dylib $(libdir)/libmagic.dylib; \
+	fi
+
 AM_CPPFLAGS = -DMAGIC='"$(MAGIC)"'
 AM_CFLAGS = $(CFLAG_VISIBILITY) @WARNINGS@
 
